test_cases/autocomplete_street_centroids.json

```diff
@@ -0,0 +1,146 @@
+{
+  "name": "autocomplete street centroids",
+  "priorityThresh": 1,
+  "endpoint": "autocomplete",
+  "tests": [
+    {
+      "id": 1,
+      "status": "pass",
+      "user": "missinglink",
+      "issue": "https://github.com/pelias/openstreetmap/issues/28",
+      "in": {
+        "sources": "osm",
+        "layers": "street",
+        "text": "rushendon furlong, england"
+      },
+      "expected": {
+        "coordinates": [ -0.636911, 51.83567 ],
```

Comment:
Not true! You can set the [distanceThresh](https://github.com/pelias/fuzzy-tester#test-case-files) value and it will fail if the actual value is outside that radius from the given coordinates. The default radius is 500 meters.